website/build_site.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+cp -r ../documentation content/
+jupyter nbconvert ../source/Mlos.Notebooks/*.ipynb --to markdown --output-dir content/notebooks
+
+cp ../*.md content/
+cp ../LICENSE content/
+cp ../README.md content/_index.md
+mv content/documentation/README.md content/documentation/_index.md
+sed -i 's/\.md/\//g' content/*.md
+sed -i 's/\.md/\//g' content/documentation/*.md
+
+
+if [ ! -d "themes/book" ]; then
+    git clone --depth 1 https://github.com/alex-shpak/hugo-book.git themes/book/
```

Comment:
Thanks for this.

Should we include a tag that we want it checked out at so that it doesn't magically change style on us at some point?